aips/passkeys.md

```diff
@@ -0,0 +1,412 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Passkeys
+author: hariria
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: <Draft | Last Call | Accepted | Final | Rejected>
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedback and reviews>
+type: Standard (Core/Framework)
+created: <12/14/2023>
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Passkey accounts
+
+## Summary
+
+This AIP proposes the first WebAuthn Authenticator for Aptos, enabling users to utilize passkeys and other WebAuthn credentials for on-chain accounts and transaction signing.
+
+[Passkeys](https://fidoalliance.org/passkeys/) are designed to replace passwords as a phishing resistant, faster, and more secure form of user authentication. When a user registers a passkey, a new [public key credential](https://www.w3.org/TR/webauthn-3/#public-key-credential) is created on their device's [authenticator](https://www.w3.org/TR/webauthn-3/#authenticator) - which holds and manages the credentials for that user. As of February 2023, on iOS 16+, Android 9+, and MacOS 13+, the passkey can then securely sync with their device's cloud provider (iCloud or Google Password Manager, for example) using end-to-end encryption to ensure that the passkey is recoverable in the event that the device is lost. When signing in with the passkey, the user is prompted to authenticate - often with a biometric based [authorization gesture](https://www.w3.org/TR/webauthn-3/#authorization-gesture) like Face ID or Touch ID - and provides a response that proves ownership of the passkey via a digital signature.
```

Comment:
```suggestion
[Passkeys](https://fidoalliance.org/passkeys/) are designed to replace passwords as a phishing resistant, faster, and more secure form of user authentication. When a user accesses a pass-key enabled site, it asks the user to register a passkey specific for that site. When a user registers a passkey, a new [public key credential](https://www.w3.org/TR/webauthn-3/#public-key-credential) is created on their device's [authenticator](https://www.w3.org/TR/webauthn-3/#authenticator). In future sessions with that website, the passkey can be used in lieu of a password to produce a digital signature that validates the identity of the user. Authenticators store passkeys behind the users device credentials, which can be accessed via biometric-based [authorization gesture](https://www.w3.org/TR/webauthn-3/#authorization-gesture) like Face ID or Touch ID.
```

This paragraph is saying too much.

Maybe talk about recovery in the open problems space rather than here:

" As of February 2023, on iOS 16+, Android 9+, and MacOS 13+, the passkey can then securely sync with their device's cloud provider (iCloud or Google Password Manager, for example) using end-to-end encryption to ensure that the passkey is recoverable in the event that the device is lost."

Reply:
There's a bit of nuance here since the authenticator is not always the device itself, it could be a password manager for example, so I will clarify this while removing the unnecessary content